### PR TITLE
Enable timesource ks command testing on RHEL-9

### PR DESCRIPTION
**Pull request by jstodola**

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

Makes me wonder if we shouldn't mark all tests by supported platforms, like here...

**Review by poncovka — APPROVED** [review]
Thanks!

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Comment by jkonecny12** [comment]
It would be great to have a system to mark all rhel-9+ or something like that. However, I guess it won't be that simple.

File: ntp-pools.sh [hunk 1]
```diff
@@ -18,6 +18,6 @@
 # Red Hat Author(s): Chris Lumens <clumens@redhat.com>
 
 # Fails in rhel8 with "Unknown command: timesource"
-TESTTYPE="time coverage fedora-only"
+TESTTYPE="time coverage fedora-only rhel-9-only"
```
Comment (rvykydal) [inline ntp-pools.sh:21]: fedora-only results in skipping of the test on rhel 9, see also https://github.com/rhinstaller/kickstart-tests/pull/541#issuecomment-882551874

**Comment by rvykydal** [comment]
> It would be great to have a system to mark all rhel-9+ or something like that. However, I guess it won't be that simple.

That could be done by using skip-on-rhel8 as suggested in https://github.com/rhinstaller/kickstart-tests/pull/541#issuecomment-882551874

**Comment by jkonecny12** [comment]
I know it is like that now but again it doesn't scale very well.

**Comment by rvykydal** [comment]
The skip-on-rhel-8 tag is ready to be used (https://github.com/rhinstaller/kickstart-tests/pull/547 is merged).

**Comment by rvykydal** [comment]
> @jstodola The skip-on-rhel-8 tag is ready to be used (#547 is merged).

Actually it has already been addressed in the PR.

**Pull request closed without merging**